src/apps/chatter/components/chatbox/chatbox.js

```diff
@@ -28,16 +33,23 @@ export const ChatBox = ({ roomId }) => {
   return (
     <div className='chatbox'>
 
-      {messages.map((entry) => <Message content={entry.text} author={entry.author} timestamp={entry.timestamp} />)}
+      {messages.map(Message)}
 
       <div className='chatbox__input-row'>
         <input id='chatbox__message-input'
+          ref={inputEl}
           type='text'
           size='100'
           value={textInput}
-          onChange={handleInputChange} />
-
-        <button className='chatbox__send-button btn' onClick={handleSend} disabled={textInput === ''} >Send</button>
+          onChange={handleInputChange}
+          onKeyUp={handleKeyUp}
+        />
+
+        <button className='chatbox__send-button btn'
+          onClick={handleSend}
+          disabled={textInput === ''}>
+          Send
+        </button>
```

Comment:
All attributes should be on the same line (looks like the input id is on the same line as the tag name), and brackets should be the same indentation.

```
        <button
          className='chatbox__send-button btn'
          onClick={handleSend}
          disabled={textInput === ''}
        >
          Send
        </button>
```

I've been meaning to find a eslint rule to automate this.